apps/hash-frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx

```diff
@@ -126,16 +126,16 @@ export const PageSidebar: FunctionComponent = () => {
         tooltipTitle: "",
```

Comment:
I think we should now make the path for 'Inbox' go to 'Actions' (since that is what in practice people will be using most, also more consistent with the new ordering)

Reply:
Good point, have fixed this in https://github.com/hashintel/hash/pull/4656/commits/d842c2a2be54af3169af4ca30efcb43d79e6031c